backend/src/main/java/se/chalmers/dat265/group1/model/dbEntities/OrderedPart.java

```diff
@@ -1,27 +1,33 @@
-package se.chalmers.dat265.group1.model;
+package se.chalmers.dat265.group1.model.dbEntities;
+
+import se.chalmers.dat265.group1.model.DataModel;
 
 public class OrderedPart extends DataModel {
 
     private int id;
     private int orderID;
-    private int amountOrdererd;
+    private int amount;
 
     public OrderedPart() {
         id = -1;
         orderID = -1;
-        amountOrdererd = -1;
+        amount = -1;
     }
 
     public int getOrderID() {
         return orderID;
     }
 
-    public int getAmountOrdererd() {
-        return amountOrdererd;
+    public int getAmount() {
+        return amount;
     }
 
     @Override
     public int getId() {
         return id;
     }
+
+    public void setID(int ID) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)